sub_cal/sub_cal/src/test/java/model/CalculatorTest.java

```diff
@@ -0,0 +1,35 @@
+package model;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CalculatorTest {
+    private static Calculator calculator = new Calculator();
+    @ParameterizedTest
+    @DisplayName("결과값 확인")
+    @MethodSource("testData")
+    void calculate(String expression,String expected) {
+        String result;
+        result = calculator.calculate(expression);
```

Comment:
```suggestion
        String result = calculator.calculate(expression);
```

혹시 줄바꿈을 한 다른 이유가 있을까요?

Reply:
줄바꿈이 굳이 필요한 것 같지않은데 제가 신경을 못쓴것 같습니다.
해당 코드 수정하였습니다 !